packages/openapi-v3/src/types.ts

```diff
@@ -43,6 +43,8 @@ export interface TagsDecoratorMetadata {
   tags: string[];
 }
 
+export type VisibilityDecoratorMetadata = 'documented' | 'undocumented';
```

Comment:
Let's call this type from the point of view of consumers (LB app developers). IMO, the fact that this value is usually provided via `@oas.visibility` decorator is an implementation detail that should not leak into our public API.

Proposed name:

```suggestion
export type OperationVisibility = 'documented' | 'undocumented';
```